jabref-template/listrefs.layout

```diff
@@ -4,27 +4,41 @@
 
 <tr id="\format{\bibtexkey}" class="entry">
   <td>
-    \begin{author}\format[Authors(FirstFirst,Initials,Comma,Comma),HTMLChars]{\author}\end{author}
```

Comment:
I'm surprised to see the {author} sections go. May I ask, what effect does this have?